### PR TITLE
SASSV1

### PR DESCRIPTION
**Pull request by WildBillDev**
Sass token commits with PW

### CODE REVIEW

**Comment by WildBillDev** [comment]
Thanks for making this easy :)

Much Love,
Wild Bill

**Comment by WildBillDev** [comment]
got this error when running build

Command failed: git log --diff-filter=A --follow --format=%aD -- C:\Users\natha\Creative Cloud Files Personal Account kickbush.ssosva@gmail.com 391F56B85FC451860A495C23@AdobeID\Sagaz\Generator\sagaz dApp build\token_Main_SASS\token-registry-SASS\tokens\main\0x0000000000000000000000000000456e65726779 | tail -1
'git' is not recognized as an internal or external command,
operable program or batch file.

**Comment by WildBillDev** [comment]
changed the SASS to sass as well let me know if  im doing something wrong on my end much love

**Comment by WildBillDev** [comment]
do i need to add the $ to this

**Comment by WildBillDev** [comment]
have looked and yes its got the $ in it am working on the push right now for some reason i thought it was not needed lol one sec

**Comment by WildBillDev** [comment]
there that should work now boy im sorry for all this over one symbol feel stupid lol

**Comment by WildBillDev** [comment]
I was going by examples in the repo

File: tokens/main/0x9b84cf29c8b5b906dd88e85fe6262f1ce1384f6b/additional.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "website":"https://thesagaz.com/", 
+    "links":{
+      "twitter":"https://twitter.com/VeSagaz",
+      "founder_twitter":"https://twitter.com/WildBillArt",
+      "Co_founder_twitter":"https://twitter.com/PaullyTV",
```
Comment (libotony) [inline tokens/main/0x9b84cf29c8b5b906dd88e85fe6262f1ce1384f6b/additional.json:6]: Only `twitter` is allowed

Comment (WildBillDev) [reply tokens/main/0x9b84cf29c8b5b906dd88e85fe6262f1ce1384f6b/additional.json:6]: k sounds good

Comment (WildBillDev) [reply tokens/main/0x9b84cf29c8b5b906dd88e85fe6262f1ce1384f6b/additional.json:6]: lol wasn't sure but figured i would try lol I am away from PC I will get this done by tonight

**Review by libotony — APPROVED** [review]

**Pull request merged**